test/controllers/static_pages_controller_test.rb

```diff
@@ -8,17 +8,13 @@ class StaticPagesControllerTest < ActionController::TestCase
   end
 
   test 'should get help' do
+    skip 'the view is not implemented!'
     get :help
     assert_response :success
   end
 
-  test 'should get feed' do
-    get :feed
-    assert_response :success
-  end
-
-  test 'should get admin' do
-    get :admin
```

Comment:
I couldn't find any trace of the `admin` action in `StaticPagesController` or in `config/routes.rb`.